pgmpy/independencies/Independencies.py

```diff
@@ -50,9 +50,16 @@ def __str__(self):
 
     __repr__ = __str__
 
+    def __eq__(self, other):
+        for self_independency in self.get_assertions():
+            if self_independency not in other.get_assertions():
+                return False
+        return True
+
     def get_assertions(self):
         """
-        Returns the independencies object which is a set of IndependenceAssertion objects.
+        Returns the independencies object which is a set of
+        IndependenceAssertion objects.
```

Comment:
@kshitij10496 I am guessing you did this because of PEP8 ? We use 120 char line limit so this is not necessary.